benchmarks/common.py

```diff
@@ -360,6 +426,12 @@ def baselines(models, model_iter_fn, example_inputs, args):
     """
     Common measurement code across all baseline experiments.
     """
+
+    if args.dynamic_shapes:
+        # TODO possibly merge ds logic back to main baselines fn, but kept it separate
+        # during exploration phase
+        return baselines_ds(models, model_iter_fn, example_inputs, args)
```

Comment:
Are we missing this function?

Reply:
oops this was leftover from my first attempt, i am cleaning it up now.